client/components/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Image } from 'semantic-ui-react';
+
+const badge = `${window.location.href}superhost_badge.png`;
+
+const badgePosition = {
+  position: 'absolute',
+  left: '47px',
+  top: '35px'
+};
+
+const Header = ({name, superhost, img, joined}) => (
+  <div style={{ display: 'flex', marginBottom: '24px'}}>
+  <Image style={{marginRight: '16px'}} circular width={56} src={img}/>
+  {superhost}
+  {superhost ? <Image height={23} src={badge} style={badgePosition} /> : null}
+  {name !== null ? <div><h2 style={{marginBottom: '0px'}}>Hosted by {name.split(' ')[0]}</h2><p style={{color: 'rgb(113, 113, 113)'}}>Joined in {joined.split(' ')[1]} {joined.split(' ')[3]} </p> </div>: null }
+  </div>
```

Comment:
will you be doing a mix of Semantic UI & Styled Components going forward or just Semantic?

Reply:
I wanted to incorporate styled components but I had a lot of issues with the installation for proxy and that wasted a good chunk of my time lol. I will think about it later. I am having so much fun learning semantics tho! Their base font is similar to airbnb's